commands/verify.go

```diff
@@ -100,16 +103,27 @@ func (v *VerifyCmd) AuthorizedKeysCommand(ctx context.Context, userArg string, t
 	if err != nil {
 		return "", err
 	}
+
 	if pkt, err := cert.VerifySshPktCert(ctx, v.PktVerifier); err != nil { // Verify the PKT contained in the cert
 		return "", err
-	} else if err := v.CheckPolicy(userArg, pkt, certB64Arg, typArg); err != nil { // Check if username is authorized
-		return "", err
-	} else { // Success!
-		// sshd expects the public key in the cert, not the cert itself. This
-		// public key is key of the CA that signs the cert, in our setting there
-		// is no CA.
-		pubkeyBytes := ssh.MarshalAuthorizedKey(cert.SshCert.SignatureKey)
-		return "cert-authority " + string(pubkeyBytes), nil
+	} else {
+		userInfo := ""
+		if accessToken := cert.GetAccessToken(); accessToken != "" {
+			if userInfoRet, err := v.UserInfoLookup(ctx, pkt, accessToken); err == nil {
+				// userInfo is optional so we should not fail if we can't access it
+				userInfo = userInfoRet
```

Comment:
would it be useful to log the `err` ?

Reply:
It would be help, but verify doesn't have a logger. Verify doesn't assume it has access to a logfile so it is worth its own PR to solve. 

Created a issue for this https://github.com/openpubkey/opkssh/issues/213